test/overviewSpec.test.js

```diff
@@ -4,19 +4,34 @@ const { findAverageRating } = require('../client/src/components/overview/Gallery
 const React = require('react');
 require('@testing-library/jest-dom');
 const {default: userEvent} = require('@testing-library/user-event');
-const { render, screen, waitFor } = require('@testing-library/react');
+const { render, screen, waitFor, fireEvent } = require('@testing-library/react');
 const { App } = require('../client/src/app.jsx');
 
-describe('Jest+RTL Workshop', function() {
+describe('FEC - Omozan overview tests', function() {
   const user = userEvent.setup();
   beforeEach(() => {
     render(<App />);
   })
+
```

Comment:
consider using what I did in my tests, which is add a div that says "loading" and then make it disappear when the initial axios request is completed. Then you can use it as the waitFor expression for the tests.